app/javascript/legacy_react/api/configuration.ts

```diff
@@ -1,6 +1,6 @@
 export class Configuration {
-    apiKey: string;
-    username: string;
-    password: string;
-    accessToken: string | (() => string);
+    apiKey!: string;
```

Comment:
Can you show that these fields will never be null? If not, you shouldn't be using a non-null assertion.